src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/PublisherFunctionsIntegrationTest.cs

```diff
@@ -38,13 +40,20 @@ public class PublisherFunctionsIntegrationTestFixture : FunctionsIntegrationTest
         .WithImage("postgres:16.1-alpine")
         .Build();
 
+    private readonly AzuriteContainer _azuriteContainer = new AzuriteBuilder()
+        .WithImage("mcr.microsoft.com/azure-storage/azurite:3.31.0")
+        .WithInMemoryPersistence()
+        .Build();
+
     public async Task DisposeAsync()
     {
+        await _azuriteContainer.DisposeAsync();
```

Comment:
We might want to add a `ClearAzureDataTableTestData(StorageConnectionString());` line to the `DisposeAsync` method in the `PublisherFunctionsIntegrationTest` class above. (I had to leave the comment here because Github won't let me leave it above :/ )

[Note, we'd have to add some methods `StorageConnectionString` to the `PublisherFunctionsIntegrationTest` class and the fixture class, just like we do in `NotifierFunctionsIntegrationTest` and `NotifierFunctionsIntegrationTestFixture`, for this to work]